test/e2e/gameserver_test.go

```diff
@@ -210,7 +219,7 @@ func TestGameServerUnhealthyAfterDeletingPod(t *testing.T) {
 	t.Parallel()
 	gs := framework.DefaultGameServer(framework.Namespace)
 	readyGs, err := framework.CreateGameServerAndWaitUntilReady(framework.Namespace, gs)
-	if err != nil {
+	if !assert.NoError(t, err) {
 		t.Fatalf("Could not get a GameServer ready: %v", err)
```

Comment:
I think we should be consistent, if we are doing a NoError, then is should have an inner t.FatalNow -- don't think it makes sense to repeat the error value twice.

Reply:
ok, I agree